ui/raidboss/data/02-arr/raid/t11.ts

```diff
@@ -159,48 +177,53 @@ export default {
     },
     {
       id: 'T11 Tether Accumulate A',
+      type: 'Tether',
       netRegex: NetRegexes.tether({ id: '001C', target: 'Kaliya' }),
       netRegexDe: NetRegexes.tether({ id: '001C', target: 'Kaliya' }),
       netRegexFr: NetRegexes.tether({ id: '001C', target: 'Kaliya' }),
       netRegexJa: NetRegexes.tether({ id: '001C', target: 'カーリア' }),
       netRegexCn: NetRegexes.tether({ id: '001C', target: '卡利亚' }),
       netRegexKo: NetRegexes.tether({ id: '001C', target: '칼리야' }),
       run: (data, matches) => {
-        data.tetherA = data.tetherA || [];
-        data.tetherA.push(matches.source);
+        (data.tetherA ??= []).push(matches.source);
```

Comment:
(Neat!)

Reply:
I had to read that twice. It makes sense now that I see it, and I'll remember the construction going forward, but I definitely wouldn't have thought of it!

Reply:
Sorry if this was confusing.  I just felt like it was simpler to make this one a one-liner!

I think this construction isn't required here.  It is equally fine to break this into two:
```typescript
        data.tetherA ??= [];
        data.tetherA.push(matches.source);
```

....but for `(data.thornMap[matches.source] ??= []).push(matches.target);` you'd have to do something like:

```typescript
        const arr = data.thornMap[matches.source] ??= [];
        // data.thornMap[matches.source] still potentially undefined here, but arr is not.
        arr.push(matches.target);
```

... I think I just had it in mind after doing the t6 changes that were similar.